### PR TITLE
golinted waddrmgr package

### PR DESCRIPTION
**Pull request by manan19**

### CODE REVIEW

**Comment by davecgh** [comment]
> waddrmgr/db.go:732: bucket.CreateBucketIfNotgitxExists undefined (type walletdb.Bucket has no field or method CreateBucketIfNotgitxExists)

File: waddrmgr/db.go [hunk 1]
```diff
@@ -729,7 +729,7 @@ func putAddrAccountIndex(tx walletdb.Tx, account uint32, addrHash []byte) error
 		return nil
 	}
 
-	bucket, err = bucket.CreateBucketIfNotExists(uint32ToBytes(account))
+	bucket, err = bucket.CreateBucketIfNotgitxExists(uint32ToBytes(account))
```
Comment (davecgh) [inline waddrmgr/db.go]: typo

Comment (manan19) [inline waddrmgr/db.go]: fixed

**Comment by davecgh** [comment]
OK

**Pull request merged**